### PR TITLE
Openbabel3

### PR DESCRIPTION
**Pull request by hvennekate**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ cmake_policy(SET CMP0020 OLD)
 project(Molsketch)
 
 # Include the extra modules (OpenBabel/uninstall)
-set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
+set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake-modules)
```
Comment (krytarowski) [inline CMakeLists.txt]: I think CMAKE_MODULE_PATH should be removed. It is overwritten by the environment.

**Comment by hvennekate** [comment]
Ok, I'll merge this now.
@krytarowski : thanks for your feedback!
@voidpin : hope that helps your builds!
@alxpl : I think we can delay a release a bit, until I've got the new atom symbol rendering ready. Or let me know when Openbabel3 becomes available on Fedora.

**Comment by voidpin** [comment]
@hvennekate 
Awesome! Thx!
@krytarowski
Can it be imported into `pkgsrc` main branch now?
Thank you both! At last a good open source 2D chemical editor :)
I've been looking for this for years.

**Comment by krytarowski** [comment]
Please release a new version of molsketch (task for @hvennekate).
Then, please upgrade the package in wip/ and we can import it to pkgsrc.

**Comment by krytarowski** [comment]
@voidpin meanwhile please check whether this version builds without any local patches in pkgsrc. So we can avoid patching locally molsketch for the next release.

**Comment by hvennekate** [comment]
@krytarowski : should I really release or do you guys want to test first? I'm a little confused: what is your definition of "release"? (Mine would be to just count the patch level up and produce some artifacts for sourceforge -- is that what you mean/need?)
And just out of curiosity: what is this pkgsrc? And how does it work?

**Comment by alxpl** [comment]
> @alxpl : I think we can delay a release a bit, until I've got the new atom symbol rendering ready. Or let me know when Openbabel3 becomes available on Fedora.

There's no rush for now, as far as I know Dominik, the OpenBabel maintainer hasn't found the time to deal with the update to version 3. Once that lands in rawhide or in a copr, I will let you know. As for me, I've just glossed over the code changes, life and PhD have been very demanding lately…

**Comment by krytarowski** [comment]
@hvennekate before importing molsketch to pkgsrc (pkgsrc.org) we need new version tagged.

It would be good to test pre-release snapshot to assure that there is no need for more patches any more.

**Comment by voidpin** [comment]
@krytarowski 
I can try building it without patches and test the package **if** I get a tag. `wip` refuses to build from master. It surely should be a work around but, I just don't know it yet.
@hvennekate 
Can you make a tarball of master and add a release tag? Something like 0.7-beta would do. No need for an official release at SourceForge, lets test this first.

**Comment by krytarowski** [comment]
@voidpin check e.g. devel/arcanist it picks a snapshot from GitHub.

**Comment by hvennekate** [comment]
@voidpin Ok, I created a tag and it appears that github automatically offers a tarball download. Is that what you mean/need?
(Also, it appears that I forgot to tag 0.6.0. Will have to do that also...)

**Comment by voidpin** [comment]
Thank you both.
I'll try building it as soon as I find some time.
@hvennekate
The 0.6.0 release is the one in SourceForge. Are you moving the project to github? Just to be sure where to look.

**Comment by hvennekate** [comment]
@voidpin Sorry, that was more of a confusing side-remark. Generally, I've used github more as a plain git repo for development and regarded SourceForge as the real publishing point.

**Comment by voidpin** [comment]
@hvennekate 
Excellent 
2020-01-18-203006_1366x768_scrot

@krytarowski 
It build without the patches.

**Comment by krytarowski** [comment]
OK, so we need now:

1. a tag of molsketch with the new version
2. an upgraded package in pkgsrc-wip
3. I can import it to pkgsrc

**Comment by voidpin** [comment]
@krytarowski
I can update pkgsrc-wip once a new version tag is available. Do you want to keep pkgsrc-users as the maintainer?

**Comment by krytarowski** [comment]
I have no opinion, but I don't want to put there myself.

**Comment by hvennekate** [comment]
Sorry to answer with such a delay. Do you still need another tag or is the last one working for you?

**Comment by voidpin** [comment]
@hvennekate
The package builds without any patches now, so thats fine.
I don't know if @krytarowski wants to pull a pre-release into the main branch, though??
I can maintain the package in the future but, I guess an official release would be to prefer. I think we might want to wait until this happens before pulling into the main branch.

**Comment by krytarowski** [comment]
I prefer to push into pkgsrc full released tag. If there is no plan to tag openbabel3 in foreseeable time, we can go with a prerelease tag.

**Comment by hvennekate** [comment]
Well, if I have a choice, I'd like to include the re-designed atom label functionality in the next full release, but I'm afraid that will take a while (by my estimates it should not be too much effort, but recently it appears to be awfully hard to allot the time for actually doing it).

**Comment by voidpin** [comment]
@hvennekate
You are the Molsketch dev here and, the choice is always yours. I can package the pre-release in the coming days and ping krytarowski when its ready. As mentioned, I can not commit to the main branch.
One question, though. If I got it correctly, the coming release will be at SourceForge but, this pre-release will exist only at GitHub? Just so I can easily apply the needed changes in the package.
Thx for your collaboration.

**Pull request merged**